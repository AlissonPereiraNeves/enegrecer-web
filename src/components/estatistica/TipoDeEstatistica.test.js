import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import TipoDeEstatistica from './TipoDeEstatistica';

describe('TipoDeEstatistica', () => {
  const propsParaTipoDeEstatistica = {
    id: 'vermelho',
    numero: '2345',
    categoria: 'Racismo',
  };

  const wrapper = shallow(<TipoDeEstatistica {...propsParaTipoDeEstatistica} />);

  it('possui a mesma estrutura', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
