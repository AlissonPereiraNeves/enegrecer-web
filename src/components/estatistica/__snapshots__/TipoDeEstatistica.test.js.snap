// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TipoDeEstatistica possui a mesma estrutura 1`] = `
<div
  id="vermelho"
>
  <h3
    className="numero"
  >
    2345
     
  </h3>
  <h6
    className="categoria"
  >
    Racismo
     
  </h6>
</div>
`;
